app/src/main/java/com/gathering/android/signup/model/AuthRepository.kt

```diff
@@ -26,6 +26,26 @@ class AuthRepository @Inject constructor() {
             }
     }
 
+    fun signInUser(email: String, pass: String, onResponseReady: (ResponseState) -> Unit) {
```

Comment:
This function is going to be added by @idaosk in another PR. You can remove this from here.

Reply:
Done!